lib/src/wizard/Wizard.tsx

```diff
@@ -180,17 +191,25 @@ const Step = styled.button`
     ${(props) => (props.disabled ? "" : "cursor: pointer")};
   }
   &:focus {
-    outline: ${(props) => props.theme.focusColor} auto 1px;
+    outline: 2px solid ${(props) => props.theme.focusColor};
   }
 `;
 
-const StepHeader = styled.div`
+type StepHeaderProps = {
+  validityIcon: boolean;
+};
+const StepHeader = styled.div<StepHeaderProps>`
```

Comment:
What do you think about removing the type and write directly `styled.div<{validityIcon: boolean}>`?

Reply:
Done!